### ISSUE TITLE
installs, but can't set as color theme

### ISSUE THREAD

**Issue by rcrooks** [brittneypostma/deep-water#2]
Hi @brittneypostma I installed this because I love the screen shots. However, there was no option to set color theme during the installation, and the theme doesn't show up in the list for the Set Color option. 

I'll try setting it in the settings JSON, but just wanted to let you know. I'm guessing maybe it's something missing the package JSON.

**Comment by brittneypostma** [comment]
Oh no! I will look at this as soon as I get back to a computer, I will let you know when it's updated!

**Comment by rcrooks** [comment]
No problem! I look forward to trying it out. 
I haven't built any VSC extensions, so not an expert on them, but I did look at the theme JSON and that all looks good. I compared the package.json to another theme that works, and see there are quite a few differences there. I am thinking maybe this part causing a problem:

`  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },`
